load-generator/src/main/java/com/amazon/opentelemetry/load/generator/emitter/OtlpMetricEmitter.java

```diff
@@ -47,43 +53,46 @@ public void emitDataLoad() throws Exception {
 
   @Override
   public void nextDataPoint() {
-    apiLatencyRecorder.record(200, Labels
-        .of(DIMENSION_API_NAME, API_NAME, DIMENSION_STATUS_CODE, "200"));
-
     apiBytesSentCounter
-        .add(100, Labels.of(DIMENSION_API_NAME, API_NAME, DIMENSION_STATUS_CODE, "200"));
+            .add(10, METRIC_ATTRIBUTES);
```

Comment:
```suggestion
            .add(10, METRIC_ATTRIBUTES);
```
Why did you change the value being added? Will this affect the tests in any way?